### PR TITLE
Feature/bidder classifications endpoint TM-1485

### PR DESCRIPTION
**Pull request by scott062**
- Maps classifications to client payload always as an array of classification ids
- Adds endpoint to access reference data for classifications (used to compare client classification payload on FE)

### CODE REVIEW

File: talentmap_api/fsbid/urls/reference.py [hunk 1]
```diff
@@ -16,6 +16,7 @@
     url(r'^codes/$', views.FSBidCodesView.as_view(), name='FSBid-skill-codes'),
     url(r'^cones/$', views.FSBidConesView.as_view(), name='FSBid-cones'),
     url(r'^locations/$', views.FSBidLocationsView.as_view(), name='FSBid-locations'),
+    url(r'^classifications/$', views.FSBidClassificationsView.as_view(), name='FSBid-classifications'),
```
Comment (mjoyce91) [inline talentmap_api/fsbid/urls/reference.py]: Do we need a classifications endpoint?

File: talentmap_api/fsbid/services/reference.py [hunk 2]
```diff
@@ -107,3 +107,10 @@ def fsbid_locations_to_talentmap_locations(data):
         "country": data.get("location_country", None),
         "is_domestic": data.get("is_domestic", None) == 1,
     }
+
+@staticmethod
+def fsbid_classifications_to_talentmap_classifications(data):
+    return {
+        "code": data.get("tp_code", None),
+        "text": data.get("tp_descr_txt", None)
```
Comment (mjoyce91) [inline talentmap_api/fsbid/services/reference.py]: add disabled_ind

Comment (scott062) [reply talentmap_api/fsbid/services/reference.py]: 🍕 Thanks! Done

File: talentmap_api/fsbid/services/reference.py [hunk 3]
```diff
@@ -112,5 +112,6 @@ def fsbid_locations_to_talentmap_locations(data):
 def fsbid_classifications_to_talentmap_classifications(data):
     return {
         "code": data.get("tp_code", None),
-        "text": data.get("tp_descr_txt", None)
+        "text": data.get("tp_descr_txt", None),
+        "disabled_ind": data.get("disabled_ind", None),
```
Comment (mjoyce91) [inline talentmap_api/fsbid/services/reference.py]: can you map these to true/false?

Comment (scott062) [reply talentmap_api/fsbid/services/reference.py]: Yes 👍

**Pull request merged**